Makefile

```diff
@@ -0,0 +1,53 @@
+DEFAULT_GOAL := help
+
+# Include .env and export it so variables set in there are available in the
+# Makefile.
+include .env
+export
+
+# Set these in the environment to override them. This is helpful for
+# development if you have file ownership problems because the user in the
+# container doesn't match the user on your host.
+USER_ID ?= 10001
+GROUP_ID ?= 10001
+
+.PHONY: help
+help:
+	$(info Available rules:)
+	$(info )
+	@fgrep -h "##" Makefile | fgrep -v fgrep | sed 's/\(.*\):.*##/\1:/' | column -t -s '|'
```

Comment:
This looks gross, but it makes this more user-friendly. It allows you to do `make` or `make help` and get help for the targets in the makefile.